helper/config.go

```diff
@@ -203,6 +213,10 @@ var newSelectionAlgoHeight int64 = 0
 
 var spanOverrideHeight int64 = 0
```

Comment:
Are those variables constants?

Reply:
No, we change their value based on the network- devnet, testnet or mainnet